src/View/View.php

```diff
@@ -493,11 +493,10 @@ public function renderLayout($content, $layout = null)
             return $this->Blocks->get('content');
         }
 
-        if (empty($content)) {
-            $content = $this->Blocks->get('content');
-        } else {
-            $this->Blocks->set('content', $content);
+        if (!empty($content)) {
```

Comment:
This changes how the method behaves with empty ('', '0', 0) as the first parameter. Previously the method would fetch the content block, now it will use the empty content.